File: python/twostones.py
""" Solve the Two Stones game on Kattis """
import sys

def main():
    """ Play the Game """
    stones = int(sys.stdin.read().strip())
    if stones % 2 == 0:
        print("Bob")
    else:
        print("Alice")


if __name__ == "__main__":
    main()

File: c/twostones.c
/* A solution to the Take Two Stones problem on open.kattis.com */
#include <stdio.h>

int main(void){
    int stones;
    while(scanf("%d", &stones) == 1){
        if ((stones % 2) == 0){
            printf("Bob\n");
        } else {
            printf("Alice\n");
        };
    }
    return 0;
}
